pkl-core/src/main/resources/org/pkl/core/errorMessages.properties

```
@@ -1126,3 +1130,45 @@ External {0} reader does not support scheme `{1}`.
 
 externalReaderAlreadyTerminated=\
 External reader process has already terminated.
+
+keywordNotAllowedHere=\
+Keyword `{0}` is not allowed here. (If you must use this name as identifier, enclose it in backticks.)
```

I think the parentheses message should just be a hint

```suggestion
Keyword `{0}` is not allowed here.

If you must use this name as identifier, enclose it in backticks.
```